src/spdx_tools/spdx/parser/rdf/graph_parsing_functions.py

```diff
@@ -140,7 +140,7 @@ def get_value_from_graph(
     # this is a helper method to cast some rdf types from graph.value() to be compatible with the
     # code that follows
     value = graph.value(subject=subject, predicate=predicate, object=_object, default=default, any=_any)
-    if value and not isinstance(value, (URIRef, Literal, BNode)):
+    if value != _any and value and not isinstance(value, (URIRef, Literal, BNode)):
```

Comment:
I don't get the `value != _any` part here. `_any` is a bool, why should it be equal to `value`?

Reply:
Yeah, sorry, mixed the values here. This needs to be `value != default`. While working on this I noticed that an error about wrong types would be logged if a `package` has no `files_analyzed` field. This is as the changes you commented below not related to the origin issue and rather a fix for the wrapper methods introduced with #563. Shall I create a separate PR for these changes?

Reply:
Yes, please do :)

Reply:
https://github.com/spdx/tools-python/pull/653 
The PR should be merged first as the test in this PR fails without these changes.